hail/Makefile

```diff
@@ -207,8 +233,59 @@ deploy: tag
 install-deps:
 	$(PIP) install -U -r python/requirements.txt -r python/dev-requirements.txt
 
-clean: clean-env
+clean: clean-env clean-docs
 	./gradlew clean
-	rm -rf build/deploy
+	rm -rf build/
 	rm -rf python/hail/hail-all-spark.jar
 	rm -rf python/README.md
+
+.PHONY: clean-docs
+clean-docs:
+	+$(MAKE) -C www clean
+
+.PHONY: docs
+docs: www install $(PYTHON_VERSION_INFO)
+	mkdir -p build
+	cp www build/www
+	mkdir -p build/docs
+	cp -R python/hail/docs build/docs
+	sed -E "s/\(hail\#([0-9]+)\)/(\[#\1](https:\/\/github.com\/hail-is\/hail\/pull\/\1))/g" \
+    < build/docs/change_log.md \
+    | pandoc -o build/docs/change_log.rst
+	+$(MAKE) -C www
```

Comment:
I don't know about make 3.81, but `+` is superfluous for `$(MAKE)` invocations for my version.

Reply:
removed